projects/plugins/jetpack/extensions/blocks/blogging-prompt/edit.js

```diff
@@ -49,6 +74,49 @@ function BloggingPromptsBetaEdit( { attributes, noticeOperations, noticeUI, setA
 			} );
 	}, [ isLoading, noticeOperations, promptId, setAttributes, setLoading ] );
 
+	// Get the tags for the post.
+	const { tags, tagIds, tagsHaveResolved } = useSelect( select => {
+		const { getEditedPostAttribute } = select( 'core/editor' );
+		const { getEntityRecords, hasFinishedResolution } = select( 'core' );
+		const _termIds = getEditedPostAttribute( 'tags' );
+
+		const query = {
+			_fields: 'id,name',
+			context: 'view',
+			include: _termIds.join( ',' ),
+			per_page: -1,
+		};
+
+		return {
+			tagIds: _termIds,
+			tags: _termIds.length ? getEntityRecords( 'taxonomy', 'post_tag', query ) : [],
```

Comment:
It looks like _termIds can be null and throw an error (I saw this when I was testing the block on a `page`, so maybe if we can prevent that, it's not an issue)
```suggestion
			tags: _termIds?.length ? getEntityRecords( 'taxonomy', 'post_tag', query ) : [],
```